### DISCUSSION TITLE
props.ts文件是怎么生成的？

### DISCUSSION THREAD

**Discussion by iwanglian** [Tencent/tdesign-miniprogram#1481] [Q&A] [answered]
最近在学习TDesign代码，也在看 gulp脚本。在组件代码里（比如 Toast) 里有个 props.ts. 文件，是从哪里来的呢？

该文件为脚本自动生成文件，请勿随意修改。

**Comment by LeeJim** [comment, marked as answer]
https://github.com/TDesignOteam/tdesign-api
